### PR TITLE
Added Lua course

### PR DESCRIPTION
**Pull request by Calvinfred123**
## What does this PR do?
Add resource

### Description
A YouTube Playlist for beginners
### Why is this valuable (or not)?
This provides a simple explanation for beginners of Lua which is used in game development 
### How do we know it's really free?
Available on a YouTube playlist
### For book lists, is it a book? For course lists, is it a course? etc.
A course for beginners.
## Checklist:
- [ X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X ] Include author(s) and platform where appropriate.
- [X ] Put lists in alphabetical order, correct spacing.
- [X ] Add needed indications (PDF, access notes, under construction).
- [ X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1169,6 +1169,7 @@
 ### Lua
 
 * [Learn Lua in 15 Minutes](http://tylerneylon.com/a/learn-lua/) - Tyler Neylon (HTML)
+* [Lua for beginners](https://youtube.com/playlist?list=PL9URkxPt-PndpZlw8m_NHh0vUBU5J-BRF&si=zLGG3YbQNKPIzHjb) - AlgoRythm
```
Comment (sohansai) [inline courses/free-courses-en.md:1172]:
```suggestion
* [Lua for beginners](https://www.youtube.com/playlist?list=PL9URkxPt-PndpZlw8m_NHh0vUBU5J-BRF) - AlgoRythm
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1169,6 +1169,7 @@
 ### Lua
 
 * [Learn Lua in 15 Minutes](http://tylerneylon.com/a/learn-lua/) - Tyler Neylon (HTML)
+* [Lua for beginners](https://www.youtube.com/playlist?list=PL9URkxPt-PndpZlw8m_NHh0vUBU5J-BRF) - AlgoRythm
```
Comment (sohansai) [inline courses/free-courses-en.md:1172]:
```suggestion
* [Lua for Beginners](https://www.youtube.com/playlist?list=PL9URkxPt-PndpZlw8m_NHh0vUBU5J-BRF) - AlgoRythm
```

Comment (sohansai) [reply courses/free-courses-en.md:1172]: I didn't notice it; my apologies.

**Comment by sohansai** [comment]
Two workflows await approval.

**Comment by Calvinfred123** [comment]
> Two workflows await approval.

I’ve had some questions about that, is that something I have to do manually or is it automatically done?

**Comment by sohansai** [comment]
Workflow approval is a manual process, crucial to prevent misuse. First-time contributors need approval by default. Get more details here: [Link](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks).

**Comment by Calvinfred123** [comment]
> Workflow approval is a manual process, crucial to prevent misuse. First-time contributors need approval by default. Get more details here: [Link](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks).

Thank you for the info, Since I am a first time contributor I presume I wait till I am approved to run the workflow checks?

**Comment by sohansai** [comment]
You should manage GitHub Actions settings for a repository. Since you're a first-time contributor, you'll need to enable them.

Follow this link for guidance: [Link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-required-approval-for-workflows-from-public-forks).

**Comment by Calvinfred123** [comment]
> You should manage GitHub Actions settings for a repository. Since you're a first-time contributor, you'll need to enable them.
> 
> Follow this link for guidance: [Link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-required-approval-for-workflows-from-public-forks).

Thanks for the link, I’ve looked at a couple different pages as well but none of the settings under action hasn’t worked. If I’m missing something, any info would be great.

**Comment by sohansai** [comment]
> > You should manage GitHub Actions settings for a repository. Since you're a first-time contributor, you'll need to enable them.
> > Follow this link for guidance: [Link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-required-approval-for-workflows-from-public-forks).
> 
> Thanks for the link, I’ve looked at a couple different pages as well but none of the settings under action hasn’t worked. If I’m missing something, any info would be great.

You can manage GitHub Actions settings for a repository by following the above link. If it doesn't work, go to your forked repository in your GitHub profile, select Settings, navigate to Code and Automation, choose Actions, then select General. At the top, you'll find Actions Permissions, select "Allow all actions and reusable workflows." That should do the trick!

**Comment by Calvinfred123** [comment]
> > > You should manage GitHub Actions settings for a repository. Since you're a first-time contributor, you'll need to enable them.
> > > Follow this link for guidance: [Link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-required-approval-for-workflows-from-public-forks).
> > 
> > 
> > Thanks for the link, I’ve looked at a couple different pages as well but none of the settings under action hasn’t worked. If I’m missing something, any info would be great.
> 
> You can manage GitHub Actions settings for a repository by following the above link. If it doesn't work, go to your forked repository in your GitHub profile, select Settings, navigate to Code and Automation, choose Actions, then select General. At the top, you'll find Actions Permissions, select "Allow all actions and reusable workflows." That should do the trick!

I currently have the "allow all actions and resuable workflows" but still nothing, Im thinking to close this PR and retry, Maybe this is bugged or something. All the other settings should be fine as well so im quite lost. Thank you for all the help

**Comment by eshellman** [comment]
yes, first time submitters need approval to run actions. You can thank bitcoin minors for that necessity. @sohansai can you confirm that you don't have ability to approve?
@Calvinfred123 once this PR gets merged, actions will run automatically. 
Thanks, and welcome to free-programming-books!

**Comment by sohansai** [comment]
Hi @eshellman, just to clarify, I don't have the ability to approve.

**Comment by eshellman** [comment]
@sohansai have added the label 'needs approval' so you can highlight these PRs for me.

**Pull request merged**